x/oracle/types/ballot.go

```diff
@@ -146,11 +146,22 @@ func NewClaim(power, mandatoryWinCount int64, recipient sdk.ValAddress) Claim {
 	}
 }
 
-// ClaimMapToSlice returns an array of sorted exchange rate ballots.
-func ClaimMapToSlice(claims map[string]Claim) []Claim {
+// ClaimMapToSlices returns an array of sorted exchange rate ballots and a second
+// array woth validators not in the includeMap filtered out.
+func ClaimMapToSlices(claims map[string]Claim, includeMap map[string]bool) ([]Claim, []Claim) {
 	c := make([]Claim, len(claims))
+	r := make([]Claim, len(includeMap))
 	i := 0
+	j := 0
 	for _, claim := range claims {
+		if _, ok := includeMap[claim.Recipient.String()]; ok {
+			r[j] = Claim{
+				Power:             claim.Power,
+				MandatoryWinCount: claim.MandatoryWinCount,
+				Recipient:         claim.Recipient,
+			}
+			j++
+		}
```

Comment:
Could we add an attribute to the Claim object? Maybe `RewardEligible`?

If this object is used elsewhere this would probably not be appropriate, but if it's just used here it might be better than doing two lists

Reply:
Yeah I'm okay with this then 👍